draft-yun-privacypass-arc.md

```diff
@@ -175,9 +175,41 @@ where `pkI_serialized` is the serialized version of `pkI` as described in [Secti
 
 # Token Challenge Requirements {#token-challenge-requirements}
 
-Clients receive challenges for tokens as described in {{AUTHSCHEME}}.
-The HTTP authentication challenge also SHOULD contain the following
-additional attribute:
+The ARC protocol uses a modified TokenChallenge structure from the one
+specified in {{AUTHSCHEME}}. In particular, the updated TokenChallenge
+structure is as follows:
+
+~~~
+struct {
+    uint16_t token_type = 0xC7D3; /* Type ARC(P-384) */
```

Comment:
This matches the ([yet-unwritten](https://datatracker.ietf.org/doc/html/draft-hendrickson-pp-attesterissuer-00#name-attester-to-issuer-request)) convention of putting token_type as the first byte.

Reply:
Isn't this specified [in the auth-scheme draft](https://ietf-wg-privacypass.github.io/base-drafts/draft-ietf-privacypass-auth-scheme.html#name-token-challenge-structure)?

```
All token challenges MUST begin with a 2-octet integer that defines the token type, in network byte order. This type indicates the issuance protocol used to generate the token and determines the structure and semantics of the rest of the structure. Values are registered in an IANA registry, [Section 6.2](https://ietf-wg-privacypass.github.io/base-drafts/draft-ietf-privacypass-auth-scheme.html#token-types). Client MUST ignore challenges with token types they do not support.[¶](https://ietf-wg-privacypass.github.io/base-drafts/draft-ietf-privacypass-auth-scheme.html#section-2.1.1-2)
```

Reply:
You are right. My bad. For the token challenge it works.